scanner/src/main/kotlin/Scanner.kt

```diff
@@ -67,13 +82,22 @@ fun scanOrtResult(
                 "result."
     }
 
-    // Add the projects as packages to scan.
+    // Determine the projects to scan as packages.
     val consolidatedProjects = Downloader.consolidateProjectPackagesByVcs(ortResult.getProjects(skipExcluded))
     val consolidatedReferencePackages = consolidatedProjects.keys.map { it.toCuratedPackage() }
 
-    val packagesToScan = (consolidatedReferencePackages + ortResult.getPackages(skipExcluded)).map { it.pkg }
-    val results = runBlocking { scanner.scanPackages(packagesToScan, outputDirectory, downloadDirectory) }
-    val resultContainers = results.map { (pkg, results) ->
+    val projectResults = runBlocking {
+        val packagesToScan = consolidatedReferencePackages.map { it.pkg }
+        projectScanner.scanPackages(packagesToScan, outputDirectory, downloadDirectory)
+    }
+
+    // Scan the packages from the ORT result.
+    val results = runBlocking {
```

Comment:
Call this `packageResults` similar to `projectResults` above.

Reply:
Done.